### ISSUE TITLE
Using lg sh or ah with rest api doesn't translate title to hebrew

### ISSUE THREAD

**Issue by roidayan** [hebcal/dotcom#38]
When using lg=ah or lg=sh the title is not translated into hebrew.
When using lg=h its fine.

the issue is maybe from line 753 in local/share/perl/site_perl/Hebcal.pm
https://github.com/hebcal/dotcom/blob/master/local/share/perl/site_perl/Hebcal.pm
the check is only for "h"
```
    if ($lang eq "h" || $lang_european{$lang}) {
        my $xsubj = Hebcal::translate_event($evt, $lang);
        if ($xsubj && $xsubj ne $subj) {
            $item{title_orig} = $subj;
            $subj = $xsubj;
        }
    }
```
looking in translate_event() the check is also for "ah" and "sh"
```
    } elsif ($lang eq "h" || $lang eq "ah" || $lang eq "sh") {
        return $evt->{hebrew};
```

**Comment by roidayan** [comment]
I actually dont see this file now and I only found it from search
https://github.com/hebcal/dotcom/blob/81a861877e0068a7a3724feb48e80684c76177f4/local/share/perl/site_perl/Hebcal.pm

so the source of the issue is different.

**Comment by mjradwin** [comment]
I believe the API always returns Hebrew as a separate field - so you should
use lg=s or lg=a only. lg=ah and lg=sh are designed for the HTML website,
not for the API.

**Comment by roidayan** [comment]
right. just the hebrew key and title key sometimes little different.
since 'h' is documented in the rest api i thought to use it.
example for candle lighting and havdalah it adds the time already to the title.
i'll use lg=s and parse the time manually and use hebrew key.
thanks.


            [1] => Array
                (
                    [title] => הדלקת נרות: 17:53
                    [hebrew] => הדלקת נרות
                    [title_orig] => Candle lighting
                    [date] => 2019-10-13T17:53:00+03:00
                    [category] => candles
                )
            [10] => Array
                (
                    [title] => הבדלה - 50 דקות: 18:54
                    [hebrew] => הבדלה - 50 דקות
                    [title_orig] => Havdalah (50 min)
                    [date] => 2019-10-19T18:54:00+03:00
                    [category] => havdalah
                )

**Comment by roidayan** [comment]
I was rethinking. even if 'h'm intended for web or other. choosing 'h' should translate the title to hebrew. like fr should translate to france. so if 'ah' and 'sh' documented as supported then it should work anyway.
the mistake is checking lang eq 'h' only while the inner callback does check every option.
i just think maybe the first check is redundant and line 753 can be removed.
calling translate_event() is doing the check and if nothing is needed returns undef which means title key wont be changed.
a user supposedly can send the lang according to "locale" like 'h' or 'fr' etc and always use key 'title'.
